PocketKit/Sources/PocketKit/Settings/SettingsView.swift

```diff
@@ -64,6 +64,13 @@ struct SettingsForm: View {
                 })
                 .textCase(nil)
 
+                Section(header: Text("App Customization").style(.settings.header)) {
+                    SettingsRowToggle(title: "Show App Badge Count") {
+                        model.isToggleAppBadge.toggle()
```

Comment:
do we still need this line?

Reply:
Hmm from my understanding, the other values are based on presenting another sheet, but in this case we're just toggling, so might not need it if we already have `appBadgeToggle`.

Reply:
okee dokee artichoke!